lang/user/id.xml

```diff
@@ -404,32 +393,48 @@ Jangan matikan sistem atau tutup aplikasi ini.</warning_saving>
 		<completed_setup>Kamu sekarang telah menyelesaikan penyiapan awal.
 Sistem Vita3K Kamu sudah siap!</completed_setup>
 		<select_language>Pilih bahasa.</select_language>
-		<install_highly_recommended>Menginstalasi kedua file firmware sangat dianjurkan.</install_highly_recommended>
+		<select_pref_path>Pilih pref path.</select_pref_path>
+		<current_emu_path>Path emulator saat ini</current_emu_path>
+		<change_emu_path>Ubah Path Emulator</change_emu_path>
+		<reset_emu_path>Path Reset Emulator</reset_emu_path>
+		<install_highly_recommended>Menginstalasi kedua file firmware sangat rekomendasikan.</install_highly_recommended>
```

Comment:
rekomendasikan -> direkomendasikan

Reply:
Just direkomendasikan More fitting bro

Reply:
> Just direkomendasikan More fitting bro

that's what I meant

Reply:
The direkomendasikan is more fitting with its contents